src/commoncode/compat.py

```diff
@@ -0,0 +1,49 @@
+#
```

Comment:
This commit is repetition of https://github.com/nexB/scancode-toolkit/pull/1534/commits/a4352b3a83129cacefb776ea40a8d7794c490b71